test/trento/application/usecases/hosts_test.exs

```diff
@@ -25,4 +29,42 @@ defmodule Trento.HostsTest do
       assert 6 = Hosts.get_all_sles_subscriptions()
     end
   end
+
+  describe "Check Selection" do
+    test "should dispatch command on Check Selection" do
+      host_id = Faker.UUID.v4()
+      selected_checks = Enum.map(0..4, fn _ -> Faker.Cat.name() end)
```

Comment:
Maybe we want to use UUIDs here (file under the usual topic: not mandatorily a uuid, but still for the sake of uniqueness)

Reply:
Uniqueness here is not strictly necessary, however I changed in favor of an uuid.
Hope it does not become _uuid all the things_ :sweat_smile:

Reply:
Oh well for tests I tend to prefer them because there's really no risk of collisions :sweat_smile: